### PR TITLE
Add test target for ParseImageView and ParseQueryAdapter

### PR DESCRIPTION
**Pull request by wangmengyan95**

### CODE REVIEW

**Comment by wangmengyan95** [comment]
Depends of #57

**Comment by wangmengyan95** [comment]
@hallucinogen could you help me review this PR, grantland is on PTO.

File: ParseLoginUI/src/androidTest/java/com/parse/ParseQueryAdapterTest.java [hunk 1]
```diff
@@ -0,0 +1,590 @@
...
+    TextView textView = new TextView(activity);
+    textView.setId(android.R.id.text1);
+    view.addView(textView);
+    ParseImageView imageView = new ParseImageView(activity);
+    imageView.setId(android.R.id.icon);
+    view.addView(imageView);
+    return view;
+  }
+
+}
```
Comment (hallucinogen) [inline ParseLoginUI/src/androidTest/java/com/parse/ParseQueryAdapterTest.java]: newline

File: ParseLoginUI/src/androidTest/java/com/parse/ParseQueryAdapterTest.java [hunk 2]
```diff
@@ -0,0 +1,590 @@
...
+    LinearLayout view = new LinearLayout(activity);
+    TextView textView = new TextView(activity);
+    textView.setId(android.R.id.text1);
+    view.addView(textView);
+    ParseImageView imageView = new ParseImageView(activity);
+    imageView.setId(android.R.id.icon);
+    view.addView(imageView);
+    return view;
+  }
+
```
Comment (hallucinogen) [inline ParseLoginUI/src/androidTest/java/com/parse/ParseQueryAdapterTest.java]: space

File: README.md [hunk 3]
```diff
@@ -31,19 +31,14 @@ To run our sample apps, you need to import this repo as a standalone Gradle proj
...
-        // If your app's project does not have the Parse Android SDK already, copy it into your
-        // project at YOUR_PROJECT_LIBS_PATH.
-        compile files('YOUR_PROJECT_LIBS_PATH/Parse-1.9.4.jar')
-        compile files('YOUR_PROJECT_LIBS_PATH/ParseFacebookUtilsV4-1.9.4.jar')
-
         // Uncomment if using Facebook Login (optional Maven dependency)
         // compile 'com.facebook.android:facebook-android-sdk:4.0.1'
+        // compile files('YOUR_PROJECT_LIBS_PATH/ParseFacebookUtilsV4-1.10.0.jar')
 
 5. Add the following to your `AndroidManifest.xml` within the `<application></application>` section.  You can see a complete example in our [sample app](https://github.com/ParsePlatform/ParseUI-Android/blob/master/ParseLoginSampleBasic/AndroidManifest.xml).
```
Comment (hallucinogen) [inline README.md]: This URL doesn't work. You need to change it into https://github.com/ParsePlatform/ParseUI-Android/blob/master/ParseLoginSampleBasic/src/main/AndroidManifest.xml

**Comment by hallucinogen** [comment]
Minor nits + broken links, otherwise LGTM. Feel free to make another PR for the links.

**Pull request merged**